services/ui-src/src/page/temporary-extension/TemporaryExtensionTypeInput.tsx

```diff
@@ -1,17 +1,26 @@
 import React from "react";
-import { Dropdown } from "@cmsgov/design-system";
+import { Dropdown, Review } from "@cmsgov/design-system";
 import { SelectOption } from "cmscommonlib";
 
 const TemporaryExtensionTypeInput: React.FC<{
   temporaryExtensionTypes: SelectOption[];
   handleOnChange: any;
-}> = ({ temporaryExtensionTypes, handleOnChange }) => {
+  temporaryExtensionType?: string;
+  disabled?: boolean;
+}> = ({
+  temporaryExtensionTypes,
+  handleOnChange,
+  temporaryExtensionType,
+  disabled,
+}) => {
   const defaultType: SelectOption = {
     label: "-- select a temporary extension type --",
     value: "",
   };
 
-  return (
+  return disabled ? (
+    <Review heading="Temporary Extension Type">{temporaryExtensionType}</Review>
```

Comment:
Should we use the passed in temporaryExtensionType to get the label version from: 
export const temporaryExtensionTypes: SelectOption[] = [
  { label: "1915(b) Temporary Extension", value: "1915(b)" },
  { label: "1915(c) Temporary Extension", value: "1915(c)" },
];
That would make it match the format used when viewing the details page. If not its not a big deal and to be honest the label value is a bit redundant but it would be nice if they were kept in sync.

Reply:
Honestly, I just thought it looked bad/redundant to have:
**Temporary Extension Type**
1915(b) Temporary Extension

Reply:
I dont disagree, I hate it too. Im guessing that is just what the original story declared for detail view. I part of me just twitches when they arent using the same source regardless of what that source is LOL...but yeah no big deal